From 5da5ee82383b203807f54210581b4e27c43ed33d Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Wed, 27 Sep 2017 23:38:49 +0200
Subject: [PATCH 4/4] activate all backport code

Signed-off-by: lyh802 <lyh802@126.com>
---
 os_dep/linux/ioctl_cfg80211.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/os_dep/linux/ioctl_cfg80211.c b/os_dep/linux/ioctl_cfg80211.c
index c20a96d..8bc4e69 100644
--- a/os_dep/linux/ioctl_cfg80211.c
+++ b/os_dep/linux/ioctl_cfg80211.c
@@ -234,7 +234,7 @@ static u8 rtw_chbw_to_cfg80211_chan_def(struct wiphy *wiphy, struct cfg80211_cha
 
 	chdef->chan = chan;
 	chdef->center_freq1 = cfreq;
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)) || defined(BUILD_OPENWRT)
 	chdef->freq1_offset = chan->freq_offset;
 #endif
 	chdef->center_freq2 = 0;
@@ -7621,7 +7621,7 @@ static void cfg80211_rtw_mgmt_frame_register(struct wiphy *wiphy,
 #else
 	struct net_device *ndev,
 #endif
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)) || defined(BUILD_OPENWRT)
 	struct mgmt_frame_regs *upd)
 #else
 	u16 frame_type, bool reg)
@@ -7641,7 +7641,7 @@ static void cfg80211_rtw_mgmt_frame_register(struct wiphy *wiphy,
 	pwdev_priv = adapter_wdev_data(adapter);
 
 #ifdef CONFIG_DEBUG_CFG80211
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)) || defined(BUILD_OPENWRT)
 	RTW_INFO(FUNC_ADPT_FMT" old_regs:%x, new_regs:%x\n", FUNC_ADPT_ARG(adapter),
 		pwdev_priv->report_mgmt, upd->interface_stypes);
 #else
@@ -7650,7 +7650,7 @@ static void cfg80211_rtw_mgmt_frame_register(struct wiphy *wiphy,
 #endif
 #endif
 
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)) || defined(BUILD_OPENWRT)
 	pwdev_priv->report_mgmt = upd->interface_stypes & (
 #ifdef not_yet
 		BIT(IEEE80211_STYPE_PROBE_REQ >> 4) |
@@ -10068,7 +10068,7 @@ static struct cfg80211_ops rtw_cfg80211_ops = {
 
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 37)) || defined(COMPAT_KERNEL_RELEASE)
 	.mgmt_tx = cfg80211_rtw_mgmt_tx,
-	#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0))
+	#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)) || defined(BUILD_OPENWRT)
 	.update_mgmt_frame_registrations = cfg80211_rtw_mgmt_frame_register,
 	#else
 	.mgmt_frame_register = cfg80211_rtw_mgmt_frame_register,
-- 
2.32.0.windows.2

